src/data/int/parity.lean

```diff
@@ -195,6 +192,19 @@ begin
   simp with parity_simps
 end
 
+@[simp, norm_cast] theorem even_coe_nat (n : ℕ) : even (n : ℤ) ↔ even n :=
+by rw [even_iff, nat.even_iff, ←int.coe_nat_zero, ←int.coe_nat_one, ←coe_nat_bit0, ←coe_nat_mod,
+  coe_nat_inj']
+
+@[simp, norm_cast] lemma odd_coe_nat {n : ℕ} : odd (n : ℤ) ↔ odd n :=
+by rw [odd_iff, nat.odd_iff, ←int.coe_nat_one, ←coe_nat_bit0, ←coe_nat_mod, coe_nat_inj']
+
+@[simp] lemma nat_abs_even {n : ℤ} : even n.nat_abs ↔ even n :=
+coe_nat_dvd_left.symm
+
+@[simp] lemma nat_abs_odd {n : ℤ} : odd n.nat_abs ↔ odd n :=
+by rw [odd_iff_not_even, nat.odd_iff_not_even, nat_abs_even]
```

Comment:
I'm not certain that these belong in this file. Was there a reason you thought these should go here specifically?

Reply:
They're also about transferring odd/even between ℕ and ℤ; I wonder where else we could put them.